pkg/constants/constants.go

```diff
@@ -81,6 +81,12 @@ const (
 	// In the Hosted mode, this namespace still exists on the managed cluster to contain
 	// necessary resources, like service accounts, roles and rolebindings.
 	KlusterletNamespaceAnnotation string = "import.open-cluster-management.io/klusterlet-namespace"
+
+	// DisableAutoImportAnnotation is used to disable the auto import feature of import controller.
+	// If present, the import controller will not apply either crds.yaml or import.yaml on
+	// the managed cluster and the bootstrap-hub-kubeconfig secret will not be updated as well in
+	// the backup-restore case.
+	DisableAutoImportAnnotation string = "import.open-cluster-management.io/disable-auto-import"
```

Comment:
will this be used by another consumer? if so better to move it to the foundation api repo